src/GNDStk/Node/test/child.test.cpp

```diff
@@ -0,0 +1,150 @@
+
+#include "catch.hpp"
+#include "GNDStk.hpp"
+using namespace njoy::GNDStk;
+
+
+// -----------------------------------------------------------------------------
+// Some child_ts
+// -----------------------------------------------------------------------------
+
+// temperature_t
+struct temperature_t {
+   // just the metadata, for this example
+   double value;
+   std::string unit;
+};
+
+inline void convert(const Node<> &n, temperature_t &temp)
+{
+   temp.value = n(meta::dvalue);
+   temp.unit  = n(meta::unit);
+}
+
+// foo_t, bar_t
+struct foo_t { std::string foo_id; };
+struct bar_t { std::string bar_id; };
+
+inline void convert(const Node<> &n, foo_t &foo)
+   { bool found; foo.foo_id = n(meta::id,found); }
+inline void convert(const Node<> &n, bar_t &bar)
+   { bool found; bar.bar_id = n(meta::id,found); }
+
+// isotope_t
+struct isotope_t {
+   std::string symbol;
+   int A;
+   const Node<> *nuclides;
+};
+
+inline void convert(const Node<> &n, isotope_t &iso)
+{
+   iso.symbol = n(meta::symbol);
+   iso.A = n(meta::A);
+   iso.nuclides = &n(child::nuclides);
+}
+
+// keywords with find::one
+auto temperature =
```

Comment:
This is why you were getting frustrated. `temperature` is defined here and you also wanted to define it in another `cpp` file.